docs/docs/deploy/omnibus/install-server.mdx

```diff
@@ -4,13 +4,13 @@ sidebar_position: 3
 ---
```

Comment:
I propose to remove this file since it's duplicating content on `/deploy/omnibus`

Reply:
Good catch!